### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by vishal5251**
fixed background problem.

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by iambami** [comment]
@vishal5251 please signoff your commit

**Comment by iambami** [comment]
@vishal5251 please check the DCO details on how to signoff commit

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This repository is used as the component for displaying Layer5 twitter feed on <
 Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>!
 
 <p style="clear:both;">
-<a href ="https://layer5.io/community/meshmates"><img alt="MeshMates" src=".github/readme/images/Layer5-MeshMentors.png" style="margin-right:10px; margin-bottom:7px;" width="28%" align="left" /></a>
+<a href ="https://layer5.io/community/meshmates"><img alt="MeshMates" src="https://user-images.githubusercontent.com/64528871/135861003-17d1d08a-d92f-48c4-a447-3a2c9cadef3e.png" style="margin-right:10px; margin-bottom:7px;" width="28%" align="left" /></a>
```
[the remark this answers is not in the document]
Comment (Zain-Muiz) [reply README.md:12]: Uhmm.. I dont see any new commit here ? Did you push ?

**Comment by leecalcote** [comment]
@vishal5251 are you able to incorporate the feedback given here?

**Comment by warunicorn19** [comment]
@leecalcote we could close this PR, issue was fixed in another PR.

**Comment by vishal5251** [comment]
> @leecalcote we could close this PR, issue was fixed in another PR.

yes it was

**Comment by leecalcote** [comment]
Thank you for confirming 👍

**Pull request closed without merging**